### ISSUE TITLE
Can you share some example for use?

### ISSUE THREAD

**Issue by neyrohub** [levzenin/no-name-engine#1]
thx)

**Comment by resistancelion** [comment]
> thx)
No, this repo is a by-product of ordered rewriting of this php hooks, so it leaks a lot and i cannot conclude for what it can be used other than learning PHP internals.

**Comment by resistancelion** [comment]
> thx)

-, репо это побочное от заказа доработки WPD, за конечное деньги уже получены и сырцы давно удалены. Опыта работы с, как и самой Delphi не имею.

-, this repo is a by-product of ordered WPD rewriting, so it leaks a lot and i cannot conclude for what it can be used other than learning PHP internals.
If your goal is so, look at wpd.pas, wpd.function.pas. _I guess WPD is for "PHP Delphi wrapper"_

**Comment by landall** [comment]
> > thx)
> 
> -, репо это побочное от заказа доработки WPD, за конечное деньги уже получены и сырцы давно удалены. Опыта работы с, как и самой Delphi не имею.
> 
> -, this repo is a by-product of ordered WPD rewriting, so it leaks a lot and i cannot conclude for what it can be used other than learning PHP internals. If your goal is so, look at wpd.pas, wpd.function.pas. _I guess WPD is for "PHP Delphi wrapper"_

what is wpd.pas?
Is it a github project?
